tools/quantize/ncnn2table.cpp

```diff
@@ -77,13 +77,32 @@ class QuantNet : public ncnn::Net
     int get_conv_names();
     int get_conv_bottom_blob_names();
     int get_conv_weight_blob_scales();
+    int get_input_names();

```

Comment:
get_input_names() defined but never called ?

Reply:
Sorry, It need be called before `ex.input`. In my other local repository, I add before 
```
    net.get_conv_names();
    net.get_conv_bottom_blob_names();
    net.get_conv_weight_blob_scales();
```

I fix it immediately.